src/utils.js

```diff
@@ -35,6 +38,40 @@ export function getAlignment(value: Symbol | void, prefix: string) {
   }
 }
 
+function getMarginSizeClassName(size: Symbol) {
+  switch (size) {
+    case MARGIN_SIZE.DOUBLE:
+      return 'Double'
+    case MARGIN_SIZE.HALF:
+      return 'Half'
+    case MARGIN_SIZE.DEFAULT:
+    // intentional fall through
+    default:
+      return ''
+  }
+}
+
+export function getMargin(value: Symbol, size: Symbol, prefix: string) {
```

Comment:
Will this function be used for items? Wondering if prefix needs to be passed in.